### DISCUSSION TITLE
How do I log into my account?

### DISCUSSION THREAD

**Discussion by LeonIsMostlyTired** [community/community#135232] [GitHub Education]
### Hi everyone,

I already have an account and I have been accepted, but I can't log in because I can't find the login link. And I am logged in to my account that has been accepted. Is there a special link or something?

**Comment by davevad93** [comment]
Hi @LeonIsMostlyTired ,

Do you mean signing in to GitHub Education? If so, here's the link to the [homepage](https://education.github.com/learner/learn).

**Comment by player29879** [comment]
ok
